src/mscorlib/Resources/Strings.resx

```diff
@@ -4382,6 +4332,12 @@
   <data name="PlatformNotSupported_RequiresW2kSP3" xml:space="preserve">
     <value>This operation is only supported on Windows 2000 SP3 or later operating systems.</value>
   </data>
+  <data name="PlatformNotSupported_SecureBinarySerialization" xml:space="preserve">
+    <value>Secure binary serialization is not supported on this platform.</value>
+  </data>
+  <data name="PlatformNotSupported_StrongNameSigning" xml:space="preserve">
+    <value>Strong name signing is not supported on this platform.</value>
```

Comment:
We may be inconsistent.  But from a grammar perspective, when it's used as an adjective, there should be a dash.  Otherwise it can be parsed as strong (name signing) rather than (strong name) signing.  Like I said, a nit.